README.md

```diff
@@ -1,4 +1,4 @@
-# PROJECT NAME
+# OOP_SchoolLibrary
 
```

Comment:
- [OPTIONAL] Good job documenting your project with a README file! However, it would be better if it's updated with the project's info.

Reply:
Dear @smunozmo, This change(Project Name) is on the features branch. This will be automatically updated when I will merge the pull request.

Reply:
Hi @RaoAkif,

sure, but I'm talking about the whole README file, it is really important to keep it updated. ✅ 

Your README file:
![Captura de pantalla 2022-06-13 a la(s) 9 03 48 p  m](https://user-images.githubusercontent.com/83717285/173478071-5c5407f9-34b9-4fbd-b7a4-32acc0be1707.png)